tests/fixtures/identity.sql

```diff
@@ -888,4 +888,5 @@ SELECT 1 LIMIT 1
 CAST(x AS INT128)
 CAST(x AS UINT128)
 CAST(x AS UINT256)
-SELECT export
\ No newline at end of file
+SELECT export
+SELECT 1 put
```

Comment:
Btw, my bad for suggesting this, it won't be an "identity" because it'd map `SELECT 1 put` to `SELECT 1 AS put`.